interface-definitions/interfaces-tunnel.xml.in

```diff
@@ -177,6 +177,25 @@
                   <help>IPv4 specific tunnel parameters</help>
                 </properties>
                 <children>
+                  <leafNode name="pmtu-discovery">
+                    <properties>
+                      <help>Path MTU Discovery parameters</help>
+                      <completionHelp>
+                        <list>disable enable</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>disable</format>
+                        <description>Disable Path MTU Discovery for tunnel</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>enable</format>
+                        <description>Enable Path MTU Discovery for tunnel (by default)</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>^(disable|enable)$</regex>
+                      </constraint>
+                    </properties>
+                  </leafNode>
```

Comment:
Why can't it be disabled by default and activated when the configuration is selected? Instead, let the user type enable or disable

Reply:
It enabled by default. (pmtudisc)

Reply:
It is recommended to refer to other configurations. It is activated by default. An option should be provided to disable it, such as:

```
set interfaces tunnel tun0 parameters ip disable-pmtu-discovery
```

Reply:
All additional tunnel options a passed as key-value, so you should define it in options.
https://github.com/vyos/vyos-1x/blob/current/python/vyos/ifconfig/tunnel.py#L73-L76
I have not found a solution, how to pass the valueless parameters into key-value options checks.
tunnel.py needs to be rewritten.

Reply:
Maybe you just need to make a judgment in `get_config()`, and it might be better to create a predetermined key value according to the configuration